AlpsSDK/ProximityHandler.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  ProximityHandler.swift
+//  AlpsSDK
+//
+//  Created by Wen on 26.10.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+import Alps
+
+class ProximityHandler {
+    
+    var contextManager: ContextManager!
+    static var immediateTrigger: [String: ProximityEvent] = [:]
+    static var nearTrigger: [String: ProximityEvent] = [:]
+    static var farTrigger: [String: ProximityEvent] = [:]
+    static var unknownTrigger: [String: ProximityEvent] = [:]
+    var immediateBeacons: [String] = []
+    var nearBeacons: [String] = []
+    var farBeacons: [String] = []
+    var unknownBeacons: [String] = []
+    var refreshTimer: Int = 60 * 1000 // timer is in milliseconds
+    
+    init(contextManager: ContextManager) {
+        self.contextManager = contextManager
+    }
+    
+    // Parsing beacons proximity relatively to the device that detect them
+    func parseBeaconsByProximity(_ beacons: [CLBeacon]) {
+        var ourBeacon: IBeaconTriple?
+        for beacon in beacons {
```

Comment:
nice place for `.forEach` etc, but once we change the proximity to filter - we will not need this method anymore.